test/unit/modules/keyboard.coffee

```diff
@@ -80,5 +80,20 @@ describe('Keyboard', ->
       expect(dom($('.ql-bold').get(0)).hasClass('ql-active')).toBe(true)
       expect(dom($('.ql-size').get(0)).value()).toBe(size)
     )
+
+    it('removeHotkey by name', ->
+      fn = @quill.getModule('keyboard').removeHotkey('BOLD')
+      expect(typeof fn).toBe('function');
```

Comment:
To properly test removal, the test should retrieve the exact function that is bound to the hotkey, call removeHotkey, and then check it was actually removed. Here, the tests pass if *any* function is returned and does not check any bindings were severed.